src/inference/src/dev/threading/istreams_executor.cpp

```diff
@@ -32,10 +32,9 @@ void IStreamsExecutor::Config::set_property(const ov::AnyMap& property) {
         if (key == ov::num_streams) {
             auto streams = value.as<ov::streams::Num>();
             if (streams == ov::streams::NUMA) {
-                _streams = 1;
+                _streams = get_num_numa_nodes();
             } else if (streams == ov::streams::AUTO) {
-                // bare minimum of streams (that evenly divides available number of cores)
-                _streams = get_default_num_streams();
+                _streams = 1;
```

Comment:
Please check if there is conformance test regression when number of streams is number of processors here

Reply:
Do not change logic with property `streams=AUTO`